tg/controllers.py

```diff
@@ -76,6 +76,35 @@ def __init__(self, model: Model, view: View, tg: Tdlib) -> None:
         self.tg = tg
         self.chat_size = 0.5
 
+    @bind(msg_handler, ["u"])
+    def user_innfo(self) -> None:
```

Comment:
can we show link to user's avatar as well?

Reply:
We know file path to profile, but it should be downloaded first I guess and it's not very user friendly to copy path to image.
If we will implement curses based UI for user info in the future then we could do this.